test/entrypoint.test.ts

```diff
@@ -348,16 +355,18 @@ describe('EntryPoint', function () {
         const badOp: UserOperation = {
           ...DefaultsForUserOp,
           nonce: TOUCH_PAYMASTER,
-          paymasterAndData: paymaster.address,
+          paymaster: paymaster.address,
+          paymasterVerificationGasLimit: 150000,
           sender: testWarmColdAccount.address
         }
         const beneficiaryAddress = createAddress()
+        const badOpPacked = packUserOp(badOp)
```

Comment:
not only here:
you use two different ways in tests.
one (like here) the "badOp" is unpacked, and you have a separate "badOpPacked"
in others (like many places below), you use "op = fillSignAndPack", which means the op is "packedUserOp", and simpler to use.
I think the latter is better: leaves less clutter (and also reduces the diff: the main change becomes  just "fillSign" replaced with "fillSignAndPack"

the only exception (if any) is if we ever create an op, use it and then modify it. in that case, we maybe keeping the "unpacked" version makes sense.